includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -236,6 +373,50 @@ private function get_video_id_from_url( $url ) {
 		return false;
 	}
 
+	/**
+	 * To get start time of youtube video in second.
+	 *
+	 * @param string $url Youtube URL.
+	 *
+	 * @return int Start time in second.
+	 */
+	private function get_start_time_from_url( $url ) {
+
+		if ( empty( $url ) ) {
+			return 0;
+		}
+
+		$start_time = 0;
+		$parsed_url = wp_parse_url( $url );
+
+		if ( ! empty( $parsed_url['query'] ) ) {
+			wp_parse_str( $parsed_url['query'], $query_vars );
+
+			if ( ! empty( $query_vars['start'] ) && 0 < (int) $query_vars['start'] ) {
+				$start_time = (int) $query_vars['start'];
+			}
+		}
+
+		if ( empty( $start_time ) && ! empty( $parsed_url['fragment'] ) ) {
```

Comment:
`$start_time` would always be `0` (thus "empty") here, no?

```suggestion
		if ( ! empty( $parsed_url['fragment'] ) ) {
```